src/components/FilterFunc/FilterFunc.tsx

```diff
@@ -0,0 +1,13 @@
+import { SortType } from '../../types/SortType';
+import { Todo } from '../../types/Todo';
+
+export function getFilter(todos: Todo[], sortField: SortType) {
```

Comment:
naming id bad

Probably you should rename it to `filter` of something like this